docs/scout.txt

```diff
@@ -0,0 +1,254 @@
+.. _laravel-scout:
+
+===========================
+Full-Text Search with Scout
+===========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: php framework, odm, code example, text search, atlas
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the Laravel Scout feature in
+your {+odm-long+} application. Scout allows you to implement full-text
+search on your Eloquent models. To learn more, see `Laravel Scout
+<https://laravel.com/docs/{+laravel-docs-version+}/scout>`__ in the
+Laravel documentation.
+
+The Scout integration for {+odm-long+} provides the following
+functionality:
+
+- Provides an abstraction to create :atlas:`Atlas Search indexes
+  </atlas-search/manage-indexes/>` from any MongoDB or SQL model.
+  
+  .. important:: Use Schema Builder to Create Search Indexes
+    
+     If your documents are already in MongoDB, create Search indexes
+     by using {+php-library+} or ``Schema`` builder methods to improve
+     search query performance. To learn more about creating Search
+     indexes, see the :ref:`laravel-as-index` section of the Atlas
+     Search guide.
+
+- Allows you to automatically replicate data from MongoDB into a
```

Comment:
It seems to me that “allows” is a word to be avoided (exept when talking about permissions, security). It's better to use "enable", for example.